### ISSUE TITLE
Webstorm plugin not formatting files. (CLI formats successfully with --write flag)

### ISSUE THREAD

**Issue by iSuslov** [biomejs/biome#3531]
**Webstorm**:
Build #WS-241.18034.50, built on June 19, 2024
**Biome plugin**:
1.2.1

Issue: when trying to run format with biome, webstorm prints "Running Biome check with ()" and nothing happens

from logs:
```
├─ 52ms ERROR biome_lsp::session Couldn't set client configuration: invalid type: null, expected struct WorkspaceSettings
```

### Environment information

```bash


CLI:
  Version:                      1.8.3
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.12.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              true
  Organize imports disabled:    false
  VCS disabled:                 true

Formatter:
  Format with errors:           true
  Indent style:                 Tab
  Indent width:                 2
  Line ending:                  Lf
  Line width:                   80
  Attribute position:           Auto
  Ignore:                       []
  Include:                      []

JavaScript Formatter:
  Enabled:                      true
  JSX quote style:              Double
  Quote properties:             AsNeeded
  Trailing commas:              All
  Semicolons:                   Always
  Arrow parentheses:            Always
  Bracket spacing:              true
  Bracket same line:            false
  Quote style:                  Double
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           Auto

JSON Formatter:
  Enabled:                      true
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Trailing Commas:              unset

CSS Formatter:
  Enabled:                      false
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Quote style:                  Double

Workspace:
  Open Documents:               0
```

### Configuration

```JSON
{
  "$schema": "node_modules/@biomejs/biome/configuration_schema.json",
  "formatter": {
    "enabled": true
  },
  "linter": {
    "enabled": false
  },
  "organizeImports": {
    "enabled": true
  }
}
```

### Playground link

https://plugins.jetbrains.com/plugin/22761-biome

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by purecopy** [comment]
This is still not working properly in 1.5.3

**Comment by denbezrukov** [comment]
@purecopy could you check if it's your case - https://github.com/biomejs/biome-intellij/issues/114#issuecomment-2569555548
